internals/search.py

```diff
@@ -0,0 +1,63 @@
+# -*- coding: utf-8 -*-
+# Copyright 2021 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License")
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from __future__ import division
+from __future__ import print_function
+
+import logging
+
+from framework import users
+from internals import models
+from internals import notifier
+
+
+def process_pending_approval_me_query():
+  """Return a list of features needing approval by current user."""
+  # TODO(jrobbins): write this
+  return []
+
```

Comment:
Should be single space between functions

Reply:
I think that two blank lines is correct in this case because these are top-level functions.  If they were methods, then it would be one line.
https://google.github.io/styleguide/pyguide.html#s3.5-blank-lines
```Two blank lines between top-level definitions, be they function or class definitions. One blank line between method definitions and between the class line and the first method.```